### PR TITLE
[HWORKS-544] move featurestore search out of epipe

### PR DESCRIPTION
**Pull request by o-alex**

### CODE REVIEW

File: include/FileProvenanceConstants.h [hunk 1]
```diff
@@ -202,94 +202,6 @@ namespace FileProvenanceConstants {
...
-        return DONT_EXIST_STR();
-      } else {
-        return "featuregroup";
-      }
-    } else {
-      return DONT_EXIST_STR();
-    }
-  }
-
   inline Int64 getFeatureViewParentIId(Ndb* conn, INodeTable inodesTable, Int64 parentIId) {
```
Comment (maismail) [inline include/FileProvenanceConstants.h:205]: Is this needed anywhere?

File: include/FileProvenanceConstants.h [hunk 2]
```diff
@@ -303,19 +215,6 @@ namespace FileProvenanceConstants {
...
-      Int64 featureViewIId = getFeatureViewParentIId(conn, inodesTable, datasetIId);
-      return FileProvenanceConstants::featureViewArtifact(parentIId, datasetIId, featureViewIId, inodeName);
-    } else if(FileProvenanceConstants::isFeaturestore(projectName, datasetName)) {
-      return FileProvenanceConstants::featureStoreArtifact(parentIId, datasetIId, inodeName);
-    } else {
-      return DONT_EXIST_STR();
-    }
-  }
-
   inline bool typeMLFeature(FileProvenanceRow row) {
```
Comment (maismail) [inline include/FileProvenanceConstants.h:218]: is this still needed?

**Review by maismail — CHANGES_REQUESTED** [review]
@o-alex can you open a PR on epipe-chef to update the version as part of this JIRA

File: include/FileProvenanceConstants.h (hunk 1 above)
Comment (o-alex) [reply include/FileProvenanceConstants.h:205]: removed

File: include/FileProvenanceConstants.h (hunk 2 above)
Comment (o-alex) [reply include/FileProvenanceConstants.h:218]: this one is still used by provenance

**Review by maismail — APPROVED** [review]

**Pull request merged**